frontend/src/static/js/api.js

```diff
@@ -137,6 +137,12 @@ window.getGroupName = function getGroupName(group, filterGroupSelection) {
   }
 };
 
+window.getGroupByAuthorDisplayName = function getGroupByAuthorDisplayName(authorRepos) {
+  return authorRepos.reduce((displayName, user) => (
+    user.displayName > displayName ? user.displayName : displayName
+  ), authorRepos[0].displayName);
+};
+
```

Comment:
This naming looks a bit weird for me. Shouldn't this be `getAuthorDisplayName` since you are returning the name but not the group?

Reply:
Thanks for the suggestion. I have changed it accordingly